scripts/rook.sh

```diff
@@ -8,6 +8,22 @@ ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
 KUBECTL_RETRY=5
 KUBECTL_RETRY_DELAY=10
 
+trap log_errors EXIT
```

Comment:
If you trap on `ERR` instead you don't need to untrap at the end IIRC.

Reply:
That looks nice, I'll use that.